src/seedu/addressbook/AddressBook.java

```diff
@@ -121,6 +122,11 @@
     private static final String COMMAND_DELETE_PARAMETER = "INDEX";
     private static final String COMMAND_DELETE_EXAMPLE = COMMAND_DELETE_WORD + " 1";
 
+    private static final String COMMAND_UPDATE_WORD = "update";
+    private static final String COMMAND_UPDATE_DESC = "Update a person's information identified by the index number used in ";
+    private static final String COMMAND_UPDATE_PARAMETER = "INDEX " + COMMAND_ADD_PARAMETERS;
```

Comment:
While doing this is not wrong, this could potentially lead to some issues in the future. For example, when you change the parameters of your add command and update _COMMAND_ADD_PARAMETERS_, you would end up changing the parameters for your update command.